### PR TITLE
load_shed: update overload manager doc to add http_downstream_filter_check

### PR DESCRIPTION
**Pull request by botengyao**
The feature was added by #33366, sry missed it.

Commit Message:
Additional Description:
Risk Level: low
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: docs/root/configuration/operations/overload_manager/overload_manager.rst [hunk 1]
```diff
@@ -170,6 +170,11 @@ The following core load shed points are supported:
       pressure, typically memory. This happens once geting data from the
       connection.
 
+  * - envoy.load_shed_points.http_downstream_filter_check
+    - Envoy will send local reply directly before creating an upstream request in
+      the router if Envoy is under pressure, typically memory. This check point
```
Comment (yanavlasov) [inline docs/root/configuration/operations/overload_manager/overload_manager.rst:175]:
```suggestion
      the router if Envoy is under resource pressure, typically memory. This change
```

**Review by yanavlasov — CHANGES_REQUESTED** [review]

File: docs/root/configuration/operations/overload_manager/overload_manager.rst (hunk 1 above)
Comment (botengyao) [reply docs/root/configuration/operations/overload_manager/overload_manager.rst]: done.

**Pull request merged**